client/src/app/(modules)/practices/(main)/(details)/[id]/page.tsx

```diff
@@ -0,0 +1,66 @@
+import { notFound } from 'next/navigation';
+
+import { ExternalLink } from 'lucide-react';
+import { Metadata } from 'next';
+
+import { getPracticesId } from '@/types/generated/practice';
+
+import { Button } from '@/components/ui/button';
+
+import Field from '../field';
+import { getPracticeFields, getPracticeImplementationFields } from '../parsers';
+
+interface PracticeDetailsProps {
+  params: { id: string };
+}
+
+export async function generateMetadata({ params }: PracticeDetailsProps): Promise<Metadata> {
+  const id = parseInt(params.id);
+  const data = await getPracticesId(id, { populate: '*' });
```

Comment:
We could maybe just fetch the title and nothing else here.

Reply:
The Id endpoint doesnt have 'fields' param but I will remove the populate